File: BAEKJOON/java/9325_얼마.java
import java.io.*;
import java.util.StringTokenizer;

public class Main {
    public static int carPrice, optionCount, eachOptionCount, optionPrice;

    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        int tc = Integer.parseInt(br.readLine());

        for (int i = 0; i < tc; i++) {
            solution();
        }

        bw.flush();
        bw.close();
    }

    private static void solution() throws IOException {
        carPrice = Integer.parseInt(br.readLine());
        optionCount = Integer.parseInt(br.readLine());

        int sum = carPrice;

        StringTokenizer st;
        for (int i = 0; i < optionCount; i++) {
            st = new StringTokenizer(br.readLine());

            eachOptionCount = Integer.parseInt(st.nextToken());
            optionPrice = Integer.parseInt(st.nextToken());

            sum += (eachOptionCount * optionPrice);
        }

        bw.write(sum + "\n");
    }
}

File: BAEKJOON/cpp/9325_얼마.cpp
#include <iostream>
#include <vector>
#include <string>
using namespace std;

int car_price, option_count, each_option_count, option_price;

void solution() {
	cin >> car_price >> option_count;

	int sum = car_price;

	for (int i = 0; i < option_count; i++) {
		cin >> each_option_count >> option_price;

		sum += (each_option_count * option_price);
	}

	cout << sum << "\n";
}

int main() {
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	int tc;
	cin >> tc;

	for (int i = 0; i < tc; i++) {
		solution();
	}

	return 0;
}